### PR TITLE
Updating readme and add countryCodes as an option for institution requests

### PR DESCRIPTION
**Pull request by jasonjia93**

### CODE REVIEW

File: src/main/java/com/plaid/client/request/InstitutionsGetRequest.java [hunk 1]
```diff
@@ -25,15 +25,23 @@ public InstitutionsGetRequest(Integer count, Integer offset) {
   }
 
   public InstitutionsGetRequest withIncludeOptionalMetadata(boolean includeOptionalMetadata) {
-    this.options = new Options(includeOptionalMetadata);
+    if (this.options == null) {
```
Comment (tanatplaid) [inline src/main/java/com/plaid/client/request/InstitutionsGetRequest.java]: wanna add `Products` to this too?

File: src/main/java/com/plaid/client/request/InstitutionsGetByIdRequest.java [hunk 2]
```diff
@@ -25,15 +26,14 @@ public InstitutionsGetByIdRequest(String institutionId) {
   }
 
   public InstitutionsGetByIdRequest withIncludeOptionalMetadata(boolean includeOptionalMetadata) {
-    this.options = new Options(includeOptionalMetadata);
```
Comment (tanatplaid) [inline src/main/java/com/plaid/client/request/InstitutionsGetByIdRequest.java]: wanna include "include_status" with this too?

File: src/test/java/com/plaid/client/integration/InstitutionsGetByIdTest.java [hunk 3]
```diff
@@ -51,7 +51,6 @@ public void testSuccessWithIncludeOptionalMetadataTrue() throws Exception {
...
...NOS8mg02oTLePLyDoWcf3oEfyM4LQB2eBgqApedUphFgIN/bWkcy1+me2EpQdintYnK4V0T6JloVi0bQ1OWxmctrictTxUIiVV5Lxdb3Ba4KssT6tn7g7RTGcb81XotJRM+Ro514lzw0pWuReXjom+CtwxnEKD2WE1m36uU339ijAOBW9sQ3wKDeXTF+r8QxdZ1oF9tWn9a/LynbYyJoePqqEPMadFvkqdlnuEPmdzjE9AsIQfd1ruq9xpKdkrK14oE4M7LQtLssrcabmvojwtIdPnBXfaJBfOW2OnDcmw04ZknZqzPmjTgmO3uQhc0tRH4LRO8NVBzfKv1IlUMDcSOhr21dBpa+yrhnkReeq5dXyJa+iOyFep02L3KtV4tNQn69lIw5wW5Fe5o0Eyfe5try9voFEJpcCpr7L8sZAA1+JKsCDwqGVT7o0Mp5pDMuy0nEyPR8fYEpogKkmp5oAsx+7IUvN6PFrHFh35LVRONQv51U5Lzavx6BBfpvXq01oKvDI4LSFT49E+vrBtHAVbNdUMfLVSE5OtNiO4MCwFHMepSAqckEGnRal5GI/2hsWTtTOmwENfxU6LEuAlyllYlpvm1hR4Z5i+52QoHp1MC3RbF80UAqfNS2NqfrBUGBzLSIezLlXObGQgHuGdI7FF4LlxqbJClqs7DYRF4ChihreIt2ShPC0iuy3pD27UumwetP8+viULOy0hQ5sgQDw6mbdmlE7bMEK5Mv/fI5m0bYSPdos3NrNkyhabhjka81VtAxiLR9pGy5UYlXjF/WzJqog7hhk5dQMYj0erdzZMVdo2rmAv4/3ZBT1U/eSlEoXatufTsPHNd1o2dli3vyW/2JQX2SoYLjdRyZy4gTa6KU/KYoubK5nTSmS0P7/RwGIb/YUtWdQ5ihxt/NQ3pS4/3SqLtjUyFyJkcAOYptO/3VzcXIOL1+SAo72itW3js3njf3w79qs/QHe8kTnrVnHzdmzjhv97rMFOeyVTWjoJ3e9srbeewFFIDbooC+NX7N47jGCmhyTM91iJ+R7EMeOjS+Z72Mt8j8eZ74FC8z2CacaHVs33mK/5How29VFy6XRHyc348L35Hlc44wMeZ3wk5owPEZ3xsatzPqh2zkf7jnAzPAz51R2+dHz0f+jg9HUXNZOIAAAAAElFTkSuQmCC", institution.getLogo());
```
Comment (ntindall) [inline src/test/java/com/plaid/client/integration/InstitutionsGetByIdTest.java]: why was this removed?

File: src/main/java/com/plaid/client/request/InstitutionsGetByIdRequest.java (hunk 2 above)
Comment (jasonjia93) [reply src/main/java/com/plaid/client/request/InstitutionsGetByIdRequest.java]: Michael was planning on handle that

File: src/test/java/com/plaid/client/integration/InstitutionsGetByIdTest.java (hunk 3 above)
Comment (jasonjia93) [reply src/test/java/com/plaid/client/integration/InstitutionsGetByIdTest.java]: This test was being flakey, I'm not totally sure why and it wasn't due to my change since I tried it out on master as well. I think it's fine just to remove it though since the url and primary color are enough to indicate extra metadata was returned

Comment (ntindall) [reply src/test/java/com/plaid/client/integration/InstitutionsGetByIdTest.java]: @jasonjia93 do we need to update the asset? Can we at least assert that it is not empty?

Comment (jasonjia93) [reply src/test/java/com/plaid/client/integration/InstitutionsGetByIdTest.java]: I'll assert that it isn't empty

Comment (jasonjia93) [reply src/test/java/com/plaid/client/integration/InstitutionsGetByIdTest.java]: ⚡️

File: src/main/java/com/plaid/client/request/InstitutionsGetByIdRequest.java (hunk 2 above)
Comment (jasonjia93) [reply src/main/java/com/plaid/client/request/InstitutionsGetByIdRequest.java]: ⚡️

File: src/main/java/com/plaid/client/request/InstitutionsGetRequest.java (hunk 1 above)
Comment (jasonjia93) [reply src/main/java/com/plaid/client/request/InstitutionsGetRequest.java]: ⚡️
